### PR TITLE
developer_manual: Update mail index format

### PR DESCRIPTION
**Pull request by sirainen**
DOV-4286

### CODE REVIEW

**Review by vmo69 — COMMENTED** [review]
Thank you, that is very interesting. I just found very little nits.

File: source/developer_manual/design/indexes/_static/mail-index-map.drawio [hunk 1]
```diff
@@ -0,0 +1 @@
...oe2gpN1hjCa30de4gs5QbnIzaObBCGFMbrteIzz3xVrsGm3orosnxVZUZ+Vj2wxx0zH5ocqzF3vRK1NumbBkr9TZEkwOmpZtEpQ5e7y/0La+BmHI7osmRD0PM5nDZi43XPcTUWxPZu5ZQYmZlwvN3rDX7uoLRvogSmzvZv+Qb3zlifTNr3u4Egmlrn7V2AX3JHed6sdaxE28c+kTuXcBnmNMIcqW8YUkfbDZ4Qxx0xsJEIfERPeUBywrOKbBKf5mMoBDS4LX4vFVKFXf4goOIHvljlPhzMLPZEOk8RS89l4EsD2SWBjJKA6U4SAMlHDtM+wLayYHEf9HOYR5OHDhd4J3WLO9GKukkObeD8fvodFif6ga6Np3kcOvtdLqYFjWEPEHHBsk0PnMNAypJB8o20DLfSTqjxv1oi3Ry1Pp/JF3mJp9kndkp1o2bYp3ZLut02cWtimNfA7SV2PjrBrJGOZ8AKhxnq9VNQznx34UoVgLKUg6UHGHwTChBnvrsnoTWWDlasmPM95jZy4hstFtvIh/ZMfrBxs52nDsGIdCVQ9iEM3hhbKG9wZ47G/KaaKTh0CJboU4aZq1Gw2pi2oOlvjimPdMyMyLAfe6yNb8grn9gqxw1jQvVPuwgHbFZs/82ZzPCEar2NN/NzGLqJWZgUikrlEjfkaBRxT3LLHPvnV5heSCtTOKGuDcseZ+lSrPrcC9bqatsUlL81G2DZKo2SOBEaV+foSnB9uv66ud4oO3WnekS/lmBn2KunuN+tgtVXY0eLx3r6TdeCP2YH6bVepcBqKSArAS5oXyXD9RV6xkS5JMo3iLC8GsS9naqysqLsvKdaFBXuMfC1cFkzWspemm15C2BHm22kg9promGVZiPrSGAV6K68u1ZIKcCjhXAIrptltut1P+WcVa+qQvkJMIR5yuv4kogr0w6tAu5nHTowjcA5/jC7eLU4U2TQ+P6myZZmu/0Vl2ndk10vaFdk8NHZS3tmmS3/8i0U5bAKy8yTW3ommcmURojiRynP04ePtsPT7ezF/uvh9k3e/79aWrLn4HdfZ7c209/zl6+PD/dz+YStdgCTotkKlpHka7Lm1IhgmHgR5yRTNcsYAU33BwEDgwn4sQqcF1+m8oUQclSs5hffNapNWRfgFXSfYXvY1RQT3+7fWHN4xeaqcqPn7mC2b8=</diagram></mxfile>
```
[the remark this answers is not in the document]
Comment (sirainen) [reply source/developer_manual/design/indexes/_static/mail-index-map.drawio:1]: I can give zoom percentage number for draw.io, not exact size. Also they have different amount of stuff in them so I'm not sure if having the exact same image size makes sense. Anyway I now exported them as 125%, is that better? They still look a little bit big to me, but maybe that's better than too small?..

[the remark this answers is not in the document]
Comment (sirainen) [reply source/developer_manual/design/indexes/_static/mail-index-map.drawio:1]: Or I guess another way might be to export the images large, but then try to add some padding on the HTML side somehow so they don't take the whole page.

[the remark this answers is not in the document]
Comment (vmo69) [reply source/developer_manual/design/indexes/_static/mail-index-map.drawio:1]: Yeah i think it's better now. 

If i get it right images default to 100% of current line width, downsizing them works as intended but when the lines get longer the pictures get bigger (up to their maximum size, seems there is no upscaling for smaller ones).

**Comment by vmo69** [comment]
It has a conflict now because it's missing commits from the  feature branch.

**Comment by sirainen** [comment]
Oh, right .. Although is the devel-manual getting good enough to be in master?

**Comment by vmo69** [comment]
Not yet i think, but i like the updated parts a lot! I think after we gave all pages a visit it should be almost good to go. Maybe we should ask all developers to take a look at some point.

There are still subtasks to be done but these can be handled by somebody else also as this was the most complicated part i think.

**Pull request merged**